src/Drupal/Commands/core/WatchdogCommands.php

```diff
@@ -34,6 +34,8 @@ class WatchdogCommands extends DrushCommands
      *   Show a listing of most recent 10 messages with a severity of notice.
      * @usage drush watchdog:show --type=php
      *   Show a listing of most recent 10 messages of type php
+     * @usage  while sleep 2; drush watchdog:show; done
+     *   Every 2 seconds, show the most recent 10 messages.
```

Comment:
Doesn't validate or work (mac osx):

![image](https://user-images.githubusercontent.com/688435/79601744-50220a00-80b7-11ea-8553-d31f3a0a78af.png)

Reply:
That was pointed out above and then fixed https://github.com/drush-ops/drush/commit/a9840accc5554d2e628a323ad179e94d2555694e

Reply:
Ah sorry! Google pointing me to old posts.